packages/core/src/sass/app/_common/layout/header.scss

```diff
@@ -382,3 +382,43 @@
         }
     }
 }
+
+.header-v2 {
+    display: flex;
+    justify-content: center;
+
+    &__acc-info {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+
+        &--top {
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            gap: 0.4rem;
+        }
+
+        &--logo {
+            height: 16px;
+            width: 16px;
+
+            svg {
```

Comment:
If you replace with quill icon, the it will be no need in lines 404-409

Reply:
replied above